modules/cli/src/main/kotlin/datamaintain/cli/App.kt

```diff
@@ -33,6 +34,8 @@ class App : CliktCommand() {
 
     private val mongoUri: String? by option(help = "mongo uri with at least database name. Ex: mongodb://localhost:27017/newName")
 
+    private val trustUri: Boolean? by option(help = "Deactivate all controls on the URI you provide Datamaintain").flag()
```

Comment:
`Disables all controls on the URI provided by Datamaintain` plutôt non ?

Reply:
De mon point de vue, Datamaintain ne donne pas d'URI, c'est l'utilisateur qui lui donne une URI lorsqu'il fait sa configuration.